src/pages/costModels/createCostModelWizard/generalInformation.tsx

```diff
@@ -139,6 +140,7 @@ class GeneralInformation extends React.Component<GeneralInformationProps> {
                     <Selector
                       label={messages.Currency}
                       direction={SelectDirection.up}
+                      appendMenuTo={() => document.body}
```

Comment:
Doesn't this break accessibility?

Reply:
It does, this is what is needed to make it work with the wizard though. Previously I was appending to body all the time, this change is to make it only do so in this case which is within the wizard.

Up to you on waiting.

Reply:
Let's wait until PatternFly has a chance to investigate on their end. Ideally, they will provide a fix that allows us to append to the parent.

I know appending to the body works with these tests. However, after meeting with the PatternFly folks, appending to the body isn't recommended for various reasons. Mainly because it effects keyboard navigation.

After further testing, I can confirm that keyboard navigation doesn't work for the Select in the wizard's general information step.

Reply:
this update is a step in that direction. i moved all but the wizard things to append to parent rather than body (this would include "add rate" and "edit rate") - so it is better than existing in that regard.

tests also look better.